packages/transfer-server/README.md

```
@@ -0,0 +1,304 @@
+The server can be configured using environment variables:
+
+| Variable | Default | Description |
+|----------|---------|-------------|
+| `PORT` | `3868` | Server listening port |
+| `CORS_ORIGINS` | `*` | Comma-separated list of allowed CORS origins |
+| `MAX_USERS_PER_ROOM` | `2` | Maximum users allowed per room |
+| `ROOM_TIMEOUT` | `3600000` | Room timeout in milliseconds (1 hour) |
+| `MAX_MESSAGE_SIZE` | `10485760` | Maximum message size in bytes (10MB) |
+
+Example `.env` file:
+```env
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)